src/test/java/utilities/Util.java

```diff
@@ -1,16 +1,27 @@
 package utilities;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
 public class Util {
 
-	/*
-	 * Sleeps for given number of seconds
-	 */
-	public static void wait(int seconds) {
-		try {
-			Thread.sleep(seconds * 1000);
-		} catch (Exception e) {
-			e.printStackTrace();
+	public static void uplad (WebDriver driver, String uploadElement) {		
+			try {
+			/*
+			 * upload file
+			 * @Author: Inhae Jeong
+			 * 
+			 */
+				WebElement upload = driver.findElement(By.xpath(uploadElement));
+				upload.sendKeys("C:/Users/Desktop/upload.png");
+				System.out.println("File is Uploaded Successfully");
+
+				Thread.sleep(2000);
+			} catch (Exception e) {
+				e.printStackTrace();
```

Comment:
1, Don't delete the existing function plz
2, The second parameter could be a `By` object to make it more reusable
3, The file that's being uploaded is hardcoded
4, I don't see the point of the hardcoded wait
5, even if you need to do a hardcoded wait, the existing function should be utilized
6, Fix formatting